### PR TITLE
Update enums-managedtenants.md

### PR DESCRIPTION
**Pull request by csgreen**
Added two new enum values for granularDelegatedAdminPrivileges and delegatedAndGranularDelegetedAdminPrivileges

### CODE REVIEW

**Comment by Lauragra** [comment]
@csgreen , please add a changelog entry for this change as well. Thanks.

**Comment by csgreen** [comment]
@isvargasmsft is there something else I need to do on this PR? I'm not sure I followed the directions properly.

**Comment by isvargasmsft** [comment]
@csgreen, you are good, please bear with us while we re-review your PR. Thank you for your work and patience.

File: api-reference/beta/resources/enums-managedtenants.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ doc_type: enumTypes
 |none|
 |delegatedAdminPrivileges|
 |unknownFutureValue|
+|granularDelegatedAdminPrivileges|
+|delegatedAndGranularDelegetedAdminPrivileges|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/enums-managedtenants.md:20]: Hi @csgreen, please add these new members to the description of the properties that have this enum as type.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @csgreen, 

My sincere apologies for the delays in getting back to you. 

Would you be so kind to add these new enums to the description of the property that uses it? 

Thank you!

File: api-reference/beta/resources/enums-managedtenants.md (hunk 1 above)
Comment (isvargasmsft) [reply api-reference/beta/resources/enums-managedtenants.md:20]: @csgreen, I identified these 2 resources with properties using this enum:
https://docs.microsoft.com/en-us/graph/api/resources/managedtenants-tenantstatusinformation?view=graph-rest-beta

https://docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Please add the new members to the properties table in these 2 topics:
https://docs.microsoft.com/en-us/graph/api/resources/managedtenants-tenantstatusinformation?view=graph-rest-beta

https://docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta

**Comment by isvargasmsft** [comment]
@csgreen gently pinging.

**Comment by idwilliams** [comment]
@isvargasmsft the review feedback should be addressed. Let @csgreen and myself know if there are any other changes that are needed.

**Review by isvargasmsft — APPROVED** [review]

**Comment by isvargasmsft** [comment]
@idwilliams @csgreen thank you for the updates. Approved. Please add the label **ready to merge** when you are ready to publish.

**Pull request merged**